.github/workflows/ci-build.yml

```diff
@@ -59,5 +80,6 @@ jobs:
 
       - run: yarn install --production=false --frozen-lockfile --ignore-scripts
       - run: yarn workspace @zooniverse/react-components build:es6
+      - run: yarn workspace @zooniverse/user build:es6
```

Comment:
don't need the fe-root build here?

Reply:
not yet, no storybooks in fe-root. so far we're just using stories in libraries that we're importing into fe-root (i.e. lib-user, shortly lib-content-pages). we might eventually have fe-root stories, then will have to add here, but not totally sure we will.